# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the . package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: .\n"
"Report-Msgid-Bugs-To: \n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: attranslate\n"
# 
# File: usr/share/build-package/github_api.py, line: 29
# File: usr/share/build-package/github_api.py, line: 73
# File: usr/share/build-package/github_api.py, line: 342
# File: usr/share/build-package/github_api.py, line: 391
#: NEEDS WORK
msgid "Could not determine repository name."
msgstr "Не можа да се определи името на хранилището."
# 
# File: usr/share/build-package/github_api.py, line: 35
#: NEEDS WORK
msgid "Could not determine current commit SHA."
msgstr "Не можа да се определи текущият SHA на комита."
# 
# File: usr/share/build-package/github_api.py, line: 44
#: NEEDS WORK
#, python-brace-format
msgid "Creating reference: {0}..."
msgstr "Създаване на справка: {0}..."
# 
# File: usr/share/build-package/github_api.py, line: 59
# File: usr/share/build-package/github_api.py, line: 65
#: NEEDS WORK
#, python-brace-format
msgid "Error creating reference: {0}"
msgstr "Грешка при създаване на референция: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 62
#: NEEDS WORK
#, python-brace-format
msgid "Reference {0} created successfully!"
msgstr "Справка {0} е създадена успешно!"
# 
# File: usr/share/build-package/github_api.py, line: 86
#: NEEDS WORK
#, python-brace-format
msgid "Could not determine SHA for {0}."
msgstr "Не можа да се определи SHA за {0}."
# 
# File: usr/share/build-package/github_api.py, line: 89
#: NEEDS WORK
#, python-brace-format
msgid "Creating branch: {0} based on {1}..."
msgstr "Създаване на клон: {0} на базата на {1}..."
# 
# File: usr/share/build-package/github_api.py, line: 101
# File: usr/share/build-package/github_api.py, line: 166
#: NEEDS WORK
#, python-brace-format
msgid "Error creating branch: {0}"
msgstr "Грешка при създаване на клон: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 102
# File: usr/share/build-package/github_api.py, line: 478
#: NEEDS WORK
#, python-brace-format
msgid "Error details: {0}"
msgstr "Детайли за грешка: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 160
#: NEEDS WORK
#, python-brace-format
msgid "Warning: Could not mark branch as non-PR: {0}"
msgstr "Предупреждение: Не можа да се маркира клонът като не-PR: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 163
#: NEEDS WORK
#, python-brace-format
msgid "Branch {0} created successfully!"
msgstr "Клон {0} е създаден успешно!"
# 
# File: usr/share/build-package/github_api.py, line: 240
#: NEEDS WORK
msgid "Failed to create branch for the build."
msgstr "Неуспешно създаване на клон за изграждането."
# 
# File: usr/share/build-package/github_api.py, line: 245
#: NEEDS WORK
#, python-brace-format
msgid "Using existing branch: {0}"
msgstr "Използване на съществуващ клон: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 267
#: NEEDS WORK
#, python-brace-format
msgid "Error retrieving remote repository URL for package: {0}"
msgstr "Грешка при извличане на URL адреса на отдалеченото хранилище за пакет: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 270
#: NEEDS WORK
#, python-brace-format
msgid "Detected repository: {0}"
msgstr "Открито хранилище: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 285
#: NEEDS WORK
msgid "Triggering build workflow on GitHub..."
msgstr "Стартиране на работен процес за изграждане в GitHub..."
# 
# File: usr/share/build-package/github_api.py, line: 294
#: NEEDS WORK
#, python-brace-format
msgid "Error triggering workflow. Response code: {0}"
msgstr "Грешка при задействане на работен поток. Код на отговор: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 297
#: NEEDS WORK
#, python-brace-format
msgid "Build workflow ({0}) triggered successfully."
msgstr "Изграждането на работен поток ({0}) беше задействано успешно."
# 
# File: usr/share/build-package/github_api.py, line: 301
#: NEEDS WORK
#, python-brace-format
msgid "URL to monitor the build: {0}"
msgstr "URL за наблюдение на изграждането: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 305
#: NEEDS WORK
#, python-brace-format
msgid "Error triggering workflow: {0}"
msgstr "Грешка при задействане на работен поток: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 313
#: NEEDS WORK
#, python-brace-format
msgid "Token file '{0}' not found."
msgstr "Файл с токен '{0}' не е намерен."
# 
# File: usr/share/build-package/github_api.py, line: 331
#: NEEDS WORK
#, python-brace-format
msgid "Token for organization '{0}' not found."
msgstr "Токенът за организация '{0}' не е намерен."
# 
# File: usr/share/build-package/github_api.py, line: 334
#: NEEDS WORK
#, python-brace-format
msgid "Error reading token: {0}"
msgstr "Грешка при четене на токен: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 345
#: NEEDS WORK
#, python-brace-format
msgid "Cleaning Actions jobs with '{0}' status..."
msgstr "Почистване на действия с работни места със статус '{0}'..."
# 
# File: usr/share/build-package/github_api.py, line: 354
#: NEEDS WORK
#, python-brace-format
msgid "Error fetching Actions jobs. Code: {0}"
msgstr "Грешка при извличане на действията. Код: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 361
#: NEEDS WORK
#, python-brace-format
msgid "No Action jobs with '{0}' status found."
msgstr "Не са намерени действия с работни места със статус '{0}'."
# 
# File: usr/share/build-package/github_api.py, line: 368
#: NEEDS WORK
#, python-brace-format
msgid "Deleting job {0}..."
msgstr "Изтриване на задача {0}..."
# 
# File: usr/share/build-package/github_api.py, line: 378
#: NEEDS WORK
#, python-brace-format
msgid "Error deleting job {0}. Code: {1}"
msgstr "Грешка при изтриване на задача {0}. Код: {1}"
# 
# File: usr/share/build-package/github_api.py, line: 380
#: NEEDS WORK
#, python-brace-format
msgid "Deleted {0} Actions jobs with '{1}' status."
msgstr "Изтрито {0} действия с '{1}' статус."
# 
# File: usr/share/build-package/github_api.py, line: 383
#: NEEDS WORK
#, python-brace-format
msgid "Error cleaning Actions jobs: {0}"
msgstr "Грешка при почистване на действията: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 394
#: NEEDS WORK
msgid "Getting tag list..."
msgstr "Получаване на списък с тагове..."
# 
# File: usr/share/build-package/github_api.py, line: 403
#: NEEDS WORK
#, python-brace-format
msgid "Error fetching tags. Code: {0}"
msgstr "Грешка при извличане на тагове. Код: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 409
#: NEEDS WORK
msgid "No tags found."
msgstr "Няма намерени тагове."
# 
# File: usr/share/build-package/github_api.py, line: 416
#: NEEDS WORK
#, python-brace-format
msgid "Deleting tag {0}..."
msgstr "Изтриване на етикет {0}..."
# 
# File: usr/share/build-package/github_api.py, line: 427
#: NEEDS WORK
#, python-brace-format
msgid "Error deleting tag {0}. Code: {1}"
msgstr "Грешка при изтриване на етикет {0}. Код: {1}"
# 
# File: usr/share/build-package/github_api.py, line: 429
#: NEEDS WORK
#, python-brace-format
msgid "Deleted {0} tags."
msgstr "Изтрито {0} тагове."
# 
# File: usr/share/build-package/github_api.py, line: 432
#: NEEDS WORK
#, python-brace-format
msgid "Error cleaning tags: {0}"
msgstr "Грешка при почистване на тагове: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 439
#: NEEDS WORK
msgid "Source branch name is required to create a pull request"
msgstr "Изисква се име на източната клонка, за да се създаде искане за сливане."
# 
# File: usr/share/build-package/github_api.py, line: 446
#: NEEDS WORK
#, python-brace-format
msgid "Token length: {0}"
msgstr "Дължина на токена: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 447
#: NEEDS WORK
#, python-brace-format
msgid "Repository name: {0}"
msgstr "Име на хранилище: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 448
#: NEEDS WORK
#, python-brace-format
msgid "Creating PR from '{0}' to '{1}'"
msgstr "Създаване на PR от '{0}' до '{1}'"
# 
# File: usr/share/build-package/github_api.py, line: 449
#: NEEDS WORK
msgid "Repository name could not be determined"
msgstr "Не можа да бъде определено името на хранилището."
# 
# File: usr/share/build-package/github_api.py, line: 453
#: NEEDS WORK
#, python-brace-format
msgid "Creating pull request from {0} to {1}..."
msgstr "Създаване на pull request от {0} към {1}..."
# 
# File: usr/share/build-package/github_api.py, line: 457
#: NEEDS WORK
#, python-brace-format
msgid "Merge {0} into {1}"
msgstr "Сливане на {0} в {1}"
# 
# File: usr/share/build-package/github_api.py, line: 458
#: NEEDS WORK
msgid "Automated PR created by build_package.py"
msgstr "Автоматизирано PR, създадено от build_package.py"
# 
# File: usr/share/build-package/github_api.py, line: 475
#: NEEDS WORK
#, python-brace-format
msgid "Failed to create PR: {0}"
msgstr "Неуспешно създаване на PR: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 476
#: NEEDS WORK
#, python-brace-format
msgid "API response status: {0}"
msgstr "Статус на отговора на API: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 486
#: NEEDS WORK
#, python-brace-format
msgid "Pull request created successfully: {0}"
msgstr "Успешно създадено искане за преглед: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 491
#: NEEDS WORK
msgid "Attempting to merge pull request automatically..."
msgstr "Опит за автоматично сливане на искане за изтегляне..."
# 
# File: usr/share/build-package/github_api.py, line: 498
#: NEEDS WORK
msgid "Pull request merged successfully"
msgstr "Заявката за изтегляне е успешно обединена"
# 
# File: usr/share/build-package/github_api.py, line: 501
#: NEEDS WORK
#, python-brace-format
msgid "Pull request created but could not be merged automatically: {0}"
msgstr "Създадено е искане за сливане, но не можа да бъде слято автоматично: {0}"
# 
# File: usr/share/build-package/github_api.py, line: 507
#: NEEDS WORK
#, python-brace-format
msgid "Error creating pull request: {0}"
msgstr "Грешка при създаване на заявка за изтегляне: {0}"
# 
# File: usr/share/build-package/main.py, line: 21
#: NEEDS WORK
msgid "Operation cancelled by user."
msgstr "Операцията е отменена от потребителя."
# 
# File: usr/share/build-package/main.py, line: 24
#: NEEDS WORK
#, python-brace-format
msgid "Unhandled error: {0}"
msgstr "Необработена грешка: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 111
#: NEEDS WORK
#, python-brace-format
msgid "Git has changes: {0} - {1}"
msgstr "Git има промени: {0} - {1}"
# 
# File: usr/share/build-package/git_utils.py, line: 114
#: NEEDS WORK
#, python-brace-format
msgid "Error checking changes: {0}"
msgstr "Грешка при проверка на промените: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 122
# File: usr/share/build-package/git_utils.py, line: 262
# File: usr/share/build-package/git_utils.py, line: 317
#: NEEDS WORK
msgid "This operation is only available in Git repositories."
msgstr "Тази операция е налична само в Git хранилища."
# 
# File: usr/share/build-package/git_utils.py, line: 142
#: NEEDS WORK
#, python-brace-format
msgid "Current branch: {0}"
msgstr "Текуща клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 165
#: NEEDS WORK
#, python-brace-format
msgid "Most recent branch identified: {0}"
msgstr "Най-новият идентифициран клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 171
#: NEEDS WORK
#, python-brace-format
msgid "Pulling latest changes from most recent branch: {0}"
msgstr "Изтегляне на последните промени от най-новия клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 182
# File: usr/share/build-package/git_utils.py, line: 197
#: NEEDS WORK
#, python-brace-format
msgid "Successfully pulled latest changes from {0}"
msgstr "Успешно изтеглени последни промени от {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 188
#: NEEDS WORK
#, python-brace-format
msgid "Failed to pull from {0}, trying current branch: {1}"
msgstr "Неуспешно изтегляне от {0}, опитвам текущия клон: {1}"
# 
# File: usr/share/build-package/git_utils.py, line: 203
#: NEEDS WORK
msgid "Failed to pull from current branch, trying dev branch"
msgstr "Неуспешно изтегляне от текущия клон, опитвам клон dev"
# 
# File: usr/share/build-package/git_utils.py, line: 212
#: NEEDS WORK
msgid "Successfully pulled latest changes from dev"
msgstr "Успешно изтеглени последни промени от dev"
# 
# File: usr/share/build-package/git_utils.py, line: 220
#: NEEDS WORK
#, python-brace-format
msgid "Error pulling changes: {0}"
msgstr "Грешка при изтегляне на промени: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 225
#: NEEDS WORK
#, python-brace-format
msgid "Unexpected error: {0}"
msgstr "Неочаквана грешка: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 255
#: NEEDS WORK
#, python-brace-format
msgid "Error finding most recent branch: {0}"
msgstr "Грешка при намиране на най-новия клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 271
#: NEEDS WORK
#, python-brace-format
msgid "Creating new branch: {0}"
msgstr "Създаване на нов клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 275
#: NEEDS WORK
msgid "Pushing new branch to remote repository..."
msgstr "Изпращане на нов клон в отдалечено хранилище..."
# 
# File: usr/share/build-package/git_utils.py, line: 278
#: NEEDS WORK
#, python-brace-format
msgid "Branch {0} created and pushed successfully!"
msgstr "Клон {0} успешно създаден и изпратен!"
# 
# File: usr/share/build-package/git_utils.py, line: 281
#: NEEDS WORK
#, python-brace-format
msgid "Error creating or pushing branch: {0}"
msgstr "Грешка при създаване или изпращане на клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 322
#: NEEDS WORK
msgid "Getting branch list..."
msgstr "Получаване на списък с клонове..."
# 
# File: usr/share/build-package/git_utils.py, line: 368
#: NEEDS WORK
#, python-brace-format
msgid "Keeping only the most recent dev branch: {0}"
msgstr "Запазване само на най-новия dev клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 373
#: NEEDS WORK
#, python-brace-format
msgid "Removing local branch: {0}"
msgstr "Премахване на локален клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 393
#: NEEDS WORK
#, python-brace-format
msgid "Error removing local branch {0}: {1}"
msgstr "Грешка при премахване на локален клон {0}: {1}"
# 
# File: usr/share/build-package/git_utils.py, line: 398
#: NEEDS WORK
#, python-brace-format
msgid "Removing remote branch: {0}"
msgstr "Премахване на отдалечен клон: {0}"
# 
# File: usr/share/build-package/git_utils.py, line: 402
#: NEEDS WORK
#, python-brace-format
msgid "Error removing remote branch {0}: {1}"
msgstr "Грешка при премахване на отдалечен клон {0}: {1}"
# 
# File: usr/share/build-package/git_utils.py, line: 404
#: NEEDS WORK
msgid "Branch cleanup completed successfully!"
msgstr "Почистването на клон завърши успешно!"
# 
# File: usr/share/build-package/git_utils.py, line: 407
#: NEEDS WORK
#, python-brace-format
msgid "Error during branch cleanup: {0}"
msgstr "Грешка по време на почистване на клон: {0}"
# 
# File: usr/share/build-package/menu_system.py, line: 45
# File: usr/share/build-package/menu_system.py, line: 59
#: NEEDS WORK
msgid "Exit"
msgstr "Изход"
# 
# File: usr/share/build-package/menu_system.py, line: 47
# File: usr/share/build-package/menu_system.py, line: 61
#: NEEDS WORK
msgid "Back"
msgstr "Назад"
# 
# File: usr/share/build-package/menu_system.py, line: 91
#: NEEDS WORK
msgid "Use [white]arrow keys ↑↓[/] to navigate, [white]Enter[/] to select, [white]ESC[/] to exit"
msgstr "Използвайте [бял]стрелки ↑↓[/] за навигация, [бял]Enter[/] за избор, [бял]ESC[/] за изход"
# 
# File: usr/share/build-package/logger.py, line: 67
#: NEEDS WORK
msgid "ERROR"
msgstr "ГРЕШКА"
# 
# File: usr/share/build-package/logger.py, line: 119
#: NEEDS WORK
msgid "Field"
msgstr "Поле"
# 
# File: usr/share/build-package/logger.py, line: 120
#: NEEDS WORK
msgid "Value"
msgstr "Стойност"
